app/src/providers/Web3Provider.tsx

```diff
@@ -17,10 +17,11 @@ interface Web3ProviderProps {
 const Web3Context = React.createContext<Web3Interface>({} as any);
 
 export const Web3Provider: React.FC<Web3ProviderProps> = ({ networkId, accounts, children }) => {
-  const web3 = new Web3(Web3.givenProvider || 'ws://127.0.0.1:8545');
+  const { web3 } = window as any;
+  const _web3 = new Web3(web3.givenProvider || 'ws://127.0.0.1:8545');
```

Comment:
We confident this works on rinkeby / heroku combo? Have had some troubles in the past

Reply:
Only put this temporarily to make it work. You'll notice on resolved version it's much better